README.md

```diff
@@ -2,7 +2,8 @@
 
 [![Build Status](https://travis-ci.org/i-like-robots/hyperons.svg?branch=master)](https://travis-ci.org/i-like-robots/hyperons) [![Coverage Status](https://coveralls.io/repos/github/i-like-robots/hyperons/badge.svg?branch=master)](https://coveralls.io/github/i-like-robots/hyperons) [![npm version](https://badge.fury.io/js/hyperons.svg)](https://badge.fury.io/js/hyperons)
 
-Renders components written in JSX to HTML without a framework, on the server or in the browser.
+### Write code in JSX. Render it to HTML. 
```

Comment:
maybe just emphasis rather than a heading? Probably good to keep the word "components" in there